src/resource.py

```diff
@@ -40,6 +40,10 @@ def get_resource_type(resource_div):
             return 'folder'
         elif group == ['activity', 'assign', 'modtype_assign', '']:
             return 'assignment'
+        elif group == ['activity', 'url', 'modtype_url', '']:
+            # TODO what to do with other types?
```

Comment:
I am also not quite sure. I would rather add them on demand. PDFs are the most important on Moodle.
Sadly the file type has to be extracted using the name of the image displayed next to the actual item so I don't see a generic way. Or do you think one can find out the file ending of the item that is referenced by the url?
If the latter would be possible then we could simply check for common file endings such as ".pdf" and ".txt".